$OpenBSD: patch-gst_tcp_gstmultihandlesink_c,v 1.1.1.1 2012/09/06 13:00:53 ajacoutot Exp $

gstmultihandlesink.c:542: error: field 'sa' has incomplete type
gstmultihandlesink.c:544: error: field 'sa_stor' has incomplete type

--- gst/tcp/gstmultihandlesink.c.orig	Wed Sep  5 12:21:21 2012
+++ gst/tcp/gstmultihandlesink.c	Wed Sep  5 12:21:36 2012
@@ -110,6 +110,7 @@
 #include "gsttcp-marshal.h"
 
 #ifndef G_OS_WIN32
+#include <sys/socket.h>
 #include <netinet/in.h>
 #endif
 
